docs/build/build-parachains.md

```diff
@@ -147,8 +147,7 @@ parallelizable objects.
 
 These could be in the form of:
 
-- System level chains (permanent chains): [leased slots](../learn/learn-auction.md),
-  [parathread pool](../learn/learn-parathreads.md)
+- System level chains (permanent chains): [leased slots](../learn/learn-auction.md)
```

Comment:
nit: auctions are/should be defunct soon, but we can remove once Polkadot upgrades to coretime

Reply:
yeah I left it there because coretime is not here yet ;)